app/src/main/java/co/icanteach/projectx/data/CacheModule.kt

```diff
@@ -0,0 +1,28 @@
+package co.icanteach.projectx.data
+
+import android.app.Application
+import androidx.annotation.NonNull
+import androidx.room.Room
+import co.icanteach.projectx.data.local.dao.PopularTVShowDao
+import co.icanteach.projectx.data.local.room.AppDatabase
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+@Module
+class CacheModule {
```

Comment:
It's name should be "DatabaseModule"

Reply:
Cache module would be better option. Our cache module will provide database or shared preferences instance